### DISCUSSION TITLE
Use controllers on PC through Network Client

### DISCUSSION THREAD

**Discussion by Jayinem** [cathery/sys-con#187] [Feature requests]
Just going to throw in my suggestion that will probably get thrown to the back of the pile and never happen :)

On Wii U there is Hidtovpad and it has Network client which allows controllers connected to PC to work on the Wii U with Hidtovpad. I would love this feature for the Switch. 

thanks

**Comment by Jayinem** [comment]
After reading other issues it does sound similar to this https://github.com/cathery/sys-con/issues/166

But they were wanting to play with a friend who doesn't have a Switch. Not sure that's possible.

**Comment by amit27592** [comment]
From what I can gather https://github.com/jakibaki/hid-mitm is what would suit your needs. It would need to be updated though to work with the latest Atmosphere. But it's a good starting point.

**Comment by Jayinem** [comment]
Yeah that's the problem it's not updated and I would have no clue how to update it myself. 

I'm not near as interested personally than I was when I first posted this I have a Tekken 6 Arcade stick + Mayflash Magic NS + Xbox 360 wireless adapter and wanted to use the stick on Switch but now I have a gaming PC. Other than Smash Bros Ultimate which would still be cool I have all the fighting games on my PC through emulators or PC games.
